### DISCUSSION TITLE
Button changes size of frame

### DISCUSSION THREAD

**Discussion by dwburger** [TomSchimansky/CustomTkinter#1137] [General]
I'm learning customtkinter.  I have this code that places a frame and a button onto the screen:
```
from tkinter import *
import customtkinter as ctk


root = ctk.CTk()
root.geometry("800x200")

frame = ctk.CTkFrame(
    root,
    width=600,
    height=100,
    fg_color = "blue")
frame.pack(
    padx=10,
    pady=10)

button = ctk.CTkButton(
    root,
    text="Start")

button.pack()

root.mainloop()
```
When I place both in the "root", the blue frame is the right size and the button is below it.  However, if I put the button into the frame by changing "root" to "frame" in the CTkButton definition, the frame's size is lost and all I see is the button.  What am I missing here???  Thanks!

**Comment by TomSchimansky** [comment]
I tjhink that's standard for the pack manager without any options. If you use place this does not happen:
```python
from tkinter import *
import customtkinter as ctk


root = ctk.CTk()
root.geometry("800x200")

frame = ctk.CTkFrame(
    root,
    width=600,
    height=100,
    fg_color = "blue")
frame.pack(
    padx=10,
    pady=10)

button = ctk.CTkButton(
    frame,
    text="Start")
button.place(x=10, y=10)

root.mainloop()
```
But I would suggest to use grid all the time because it keeps the app responsive.
